drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/emergency_shelter_allowance_grant_loader.rb

```diff
@@ -0,0 +1,73 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# matriculation to new platform
+module HmisExternalApis::AcHmis::Importers::Loaders
+  class EmergencyShelterAllowanceGrantLoader < CustomDataElementBaseLoader
+    def filename
+      'EmergencyShelterAllowanceGrant.csv'
+    end
+
+    protected
+
+    def cde_definitions_keys
+      [
+        :esg_allowance_grant_referred,
+        :esg_allowance_grant_received,
+        :esg_allowance_grant_received_amount,
+        :esg_allowance_grant_reason_not_referred,
+      ]
+    end
+
+    def build_records
+      owner_id_by_enrollment_id = Hmis::Hud::Enrollment
+        .where(data_source: data_source)
+        .pluck(:enrollment_id, :id)
+        .to_h
+      rows.flat_map do |row|
+        enrollment_id = row_value(row, field: 'ENROLLMENTID')
```

Comment:
Should we pass this flag so we get an error if its missing?
```suggestion
        enrollment_id = row_value(row, field: 'ENROLLMENTID', required: true)
```

Reply:
nevermind I realized its the default 😁

Reply:
So `required: true` is actually the default for BaseLoader.row_value(). I made it the default since it seemed like most fields were required in the spec.